database/wiiu/accept_friend_request_and_return_friend_info.go

```diff
@@ -3,21 +3,19 @@ package database_wiiu
 import (
 	"time"
 
-	"github.com/PretendoNetwork/friends-secure/database"
-	"github.com/PretendoNetwork/friends-secure/globals"
+	"github.com/PretendoNetwork/friends/database"
+	"github.com/PretendoNetwork/friends/globals"
 	"github.com/PretendoNetwork/nex-go"
 	friends_wiiu_types "github.com/PretendoNetwork/nex-protocols-go/friends-wiiu/types"
-	"github.com/gocql/gocql"
 )
 
-func AcceptFriendRequestAndReturnFriendInfo(friendRequestID uint64) *friends_wiiu_types.FriendInfo {
+func AcceptFriendRequestAndReturnFriendInfo(friendRequestID uint64) (*friends_wiiu_types.FriendInfo, error) {
```

Comment:
Not your fault, but since we're here. This exported function is missing a comment

This happens elsewhere too but I don't want to spam this PR with tons of duplicate reviews